src/Reflection/Adapter/ReflectionMethod.php

```diff
@@ -34,6 +37,27 @@ public function __construct(private BetterReflectionMethod $betterReflectionMeth
         unset($this->class);
     }
 
+    public static function createFromMethodName(string $method): static
+    {
+        if (! str_contains($method, '::')) {
+            throw new CoreReflectionException('Argument #1 ($method) must be a valid method name');
+        }
+
+        [$className, $methodName] = explode('::', $method);
```

Comment:
What happens if a value like `foo::bar::baz` is given? I think we should have `2` as third argument here

Reply:
Added 👍